### PR TITLE
Assignment1

### PR DESCRIPTION
**Pull request by shweshipu**
Issues 1 and 2 were solved

### CODE REVIEW

File: PrincessBrideTrivia/PrincessBrideTrivia/Program.cs [hunk 1]
```diff
@@ -24,7 +24,7 @@ public static void Main(string[] args)
 
         public static string GetPercentCorrect(int numberCorrectAnswers, int numberOfQuestions)
         {
-            return (numberCorrectAnswers / numberOfQuestions * 100) + "%";
+            return ((double)numberCorrectAnswers / (double)numberOfQuestions * 100) + "%";
```
Comment (mmoran314) [inline PrincessBrideTrivia/PrincessBrideTrivia/Program.cs:27]: Pull Request title includes "Assignment 1" in the title ✔
Pull Request targets Assignment 1 branch ✔
Issue 1: Application no longer crashes ✔
Issue 2: Unit test properly passes ✔
I'm not sure if it is something worth changing, but changing the arguments in the GetPercentCorrect method to doubles will take away the need to use casting on the variables in the percent correct calculation.
Other than that, it looks great and I do not see anything that would need to be changed.

**Review by mmoran314 — CHANGES_REQUESTED** [review]
Pull Request title includes "Assignment 1" in the title ✔
Pull Request targets Assignment 1 branch ✔
Issue 1: Application no longer crashes ✔
Issue 2: Unit test properly passes  ✔

File: PrincessBrideTrivia/PrincessBrideTrivia/Program.cs (hunk 1 above)
Comment (Joshua-Pedersen) [inline PrincessBrideTrivia/PrincessBrideTrivia/Program.cs:27]: You could simplify this by just casting once. A ```double``` divided by an ```int``` will result in a ```double```
~~~
return ((double)numberCorrectAnswers / numberOfQuestions * 100) + "%";
~~~

File: PrincessBrideTrivia/PrincessBrideTrivia/Program.cs [hunk 2]
```diff
@@ -89,6 +89,8 @@ public static Question[] LoadQuestions(string filePath)
                 question.Answers[1] = answer2;
                 question.Answers[2] = answer3;
                 question.CorrectAnswerIndex = correctAnswerIndex;
+
+                questions[i] = question;
```
Comment (Joshua-Pedersen) [inline PrincessBrideTrivia/PrincessBrideTrivia/Program.cs:93]: Not sure if they are requiring this, but the added code isn't covered by a test. Could create a test to confirm the ```question``` is being placed into the ```questions``` array

**Review by Joshua-Pedersen — COMMENTED** [review]
Pull Request title includes "Assignment 1" in the title ✔
Pull Request targets Assignment 1 branch ✔
Issue 1: Application no longer crashes ✔
Issue 2: Unit test properly passes ✔

Changes clear and working.

**Pull request closed without merging**